src/main/resources/standardTestCases/array_and_map_indexing/array_and_map_indexing.wdl

```diff
@@ -0,0 +1,28 @@
+task echo_str {
+  String s
+  command { echo ${s} }
+  output { String o = read_string(stdout()) }
+  runtime { 
+   docker: "ubuntu:latest"
+  }
+}
+
+task echo_int {
+  Int i
+  command { echo ${i} }
+  output { Int o = read_int(stdout()) }
+  runtime {
+   docker: "ubuntu:latest"
+  }
+}
+
+workflow test {
+  Map[String, Int] m = {"a": 0, "b": 1, "c": 2}
+  Array[String] a = ["foo", "bar", "baz"]
```

Comment:
ToL: This could be an Array[Int] so that only the `echo_int` task is required

(Just to make it a little quicker to work out what's going on when reading the WDL)